Snakemake/workflow/profile/cluster.json

```diff
@@ -13,8 +13,8 @@
  },
  "phase":
   {
-    "mem": "150G",
-    "threads": "20",
+    "mem": "350G",
```

Comment:
In here, just make sure you have a "G" next to the number for mem - otherwise it will complain